utilities/test_suite/README.md

```diff
@@ -4,16 +4,6 @@ This repository contains four test suites for the RPP library: `image`/`voxel`/`
 
 ## Prerequisites
 
-* OpenMP
-  ```shell
-  sudo apt install libomp-dev
```

Comment:
We are removing explicitly stating OpenMP in the deps?

Reply:
Yes, we will use AOMP - AMD provided OpenMP which will get installed with ROCm